l10n_ar_edi_ux/models/l10n_ar_boarding_permission.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class L10nArBoardingPermission(models.Model):
+    _name = "l10n_ar.boarding_permission"
+    _description = 'Boarding Permission'
+    _rec_name = 'l10n_ar_boarding_permission_dst_country'
+
+    l10n_ar_boarding_permission_number = fields.Char(string="Permiso de embarque")
+    l10n_ar_boarding_permission_dst_country = fields.Many2one('res.country', string="País donde se aplica permiso de embarque")
```

Comment:
not need to add  l10n_ar_boarding_ as a prefix to all fields.

The rule for the fields name is:
- if it is a new model, one that have l10n_ar prefix then the fields does not have to have l10n_ar preffix
- If we add a method to an existing model, a generic one (does not have any l10n_ar preffix in the model name= then we need to add the l10n_ar preffix.

Also:
1. the dst_country string should be the field help instead
2. make both fields required

```suggestion
    number = fields.Char(string="Permiso de embarque", required=True)
    dst_country = fields.Many2one('res.country', string="Pais Origen", help="País donde se aplica permiso de embarque", required=True)
```